### PR TITLE
rename `NetworkSettingsOptimismAndArbitrum` > `NetworkSettingsOptimism ` for unambiguity

### LINKED ISSUE #2360: Rename `NetworkSettingsSelectOptimismAndArbitrum` component to `NetworkSettingsSelectOptimism`
_Originally posted by @jagodarybacka in https://github.com/tallycash/extension/pull/2287#discussion_r983296465_

### PR DESCRIPTION
**Pull request by greg-nagy**
We have decided to use the legacy networks select panel for gas setting on arbitrum, so it doesn't make sense anymore to keep this nameing. 

Closes #2360

### CODE REVIEW

File: ui/components/NetworkFees/NetworkSettingsSelectOptimism.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ import SharedButton from "../Shared/SharedButton"
 
 export default function NetworkSettingsOptimismAndArbitrum(): ReactElement {
```
Comment (jagodarybacka) [inline ui/components/NetworkFees/NetworkSettingsSelectOptimism.tsx:12]:
```suggestion
export default function NetworkSettingsOptimism(): ReactElement {
```

File: ui/components/NetworkFees/NetworkSettingsChooser.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ import {
 } from "@tallyho/tally-background/redux-slices/selectors/transactionConstructionSelectors"
 import { useBackgroundDispatch, useBackgroundSelector } from "../../hooks"
 import NetworkSettingsSelect from "./NetworkSettingsSelect"
-import NetworkSettingsOptimismAndArbitrum from "./NetworkSettingsSelectOptimismAndArbitrum"
+import NetworkSettingsOptimismAndArbitrum from "./NetworkSettingsSelectOptimism"
```
Comment (jagodarybacka) [inline ui/components/NetworkFees/NetworkSettingsChooser.tsx:14]:
```suggestion
import NetworkSettingsOptimism from "./NetworkSettingsSelectOptimism"
```

**Review by jagodarybacka — CHANGES_REQUESTED** [review]
2 more places to change :P

**Comment by greg-nagy** [comment]
> 2 more places to change :P

fixed, ty

**Comment by greg-nagy** [comment]
> Puzzled... The PR and first commit say we're renaming NetworkSettingsSelect but that component looks untouched?

fixed the PR title and commit msg (had to rebase)

**Review by 0xDaedalus — APPROVED** [review]
📈

**Pull request merged**